src/react/deckgl.js

```diff
@@ -138,23 +138,49 @@ export default class DeckGL extends React.Component {
       events: {
         click: this._onClick,
         mousemove: this._onMouseMove,
-        dragstart: this._onDragEvent,
-        dragmove: this._onDragEvent,
-        dragend: this._onDragEvent,
-        dragcancel: this._onDragCancel
+        panstart: this._onDragEvent,
+        panmove: this._onDragEvent,
+        panend: this._onDragEvent,
+        pancancel: this._onDragEvent
       }
     });
   }
 
+  // Get mouse position {x, y} relative to the container
+  _getPos(event) {
+    const {srcEvent, center, rootElement} = event;
+    let x;
+    let y;
+
+    if (center) {
+      // processed pointer input, supports touch
+      x = center.x;
+      y = center.y;
+    } else if (srcEvent) {
+      // fallback
+      x = srcEvent.clientX;
+      y = srcEvent.clientY;
+    }
+    if (!Number.isFinite(x)) {
```

Comment:
Should this just be an `else` statement rather than and `if isFinite`?

If one of the two loops above can generated a bad `x`, maybe add a comment why this happens.